runtime/command-version/src/main/java/io/aklivity/zilla/runtime/command/version/internal/airline/ZillaVersionCommand.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2021-2023 Aklivity Inc.
+ *
+ * Aklivity licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package io.aklivity.zilla.runtime.command.version.internal.airline;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+import com.github.rvesse.airline.annotations.Command;
+
+import io.aklivity.zilla.runtime.command.ZillaCommand;
+
+@Command(name = "version", description = "Print Zilla version")
+public final class ZillaVersionCommand extends ZillaCommand
+{
+    public ZillaVersionCommand()
+    {
+    }
+
+    @Override
+    public void run()
+    {
+        Properties properties = new Properties();
+        try (InputStream input = ZillaVersionCommand.class.getClassLoader().getResourceAsStream("version.properties"))
```

Comment:
The overall approach to embed the version in a properties file inside the JAR should not be necessary.

See `Class<?>.getModule().getModuleDescriptor().rawVersion()` instead.

Note: during non-modular execution of the code, such as during unit testing or integration testing, `Class<?>.getModule()` will return `null`.

We can potentially use `ZillaVersionCommand.class` itself as the anchor class to lookup the module, descriptor and version.